### DISCUSSION TITLE
How's the NPU support? Does Jan utilize NPUs?

### DISCUSSION THREAD

**Discussion by ux-han** [janhq/jan#3523] [Get Help] [answered]
NPUs support?

**Comment by ux-han** [comment, marked as answer]
We have an experimental implementation of ONNX, but are having some issues targeting the Snapdragon NPU.
**Current Status**

- The current status is that it works (?) somewhat with the Adreno GPU, but we haven't gotten it to work on the NPU
- We are currently busy implementing the Cortex CLI, but should be done in sprint 19 (6 August 2024)

**Reply by Neard-AI** [reply]
Will Intell Ultra processor NPU be supported soon?

**Reply by iMonZ** [reply]
> Will Intell Ultra processor NPU be supported soon?

Any updates here?

**Comment by Code-Vlue** [comment]
Any news on NPU support?

**Comment by h-n-a** [comment]
Any plans to support AMD NPUs? AMD Ryzen 7 AI 350.

```
user@pop-os:~$ readlink -f /sys/class/accel/accel0/device/driver
/sys/bus/pci/drivers/amdxdna
user@pop-os:~$ cat /sys/class/accel/accel0/device/uevent
DRIVER=amdxdna
PCI_CLASS=118000
PCI_ID=1022:17F0
PCI_SUBSYS_ID=1043:20CF
PCI_SLOT_NAME=0000:04:00.1
MODALIAS=pci:v00001022d000017F0sv00001043sd000020CFbc11sc80i00
```

**Reply by SuperCowProducts** [reply]
Same thing I was wondering...
